docs/docs/tutorials/index.md

```diff
@@ -35,6 +35,7 @@ Welcome to DSPy tutorials! We've organized our tutorials into three main categor
     - [Saving and Loading](/tutorials/saving/)
     - [Deployment](/tutorials/deployment/)
     - [Debugging & Observability](/tutorials/observability/)
+    - [Tracking DSPy Optimizers](/tutorials/optimizer_tracking/)
```

Comment:
a minor thing - for the image attached, can we make it smaller by using some image compression tool?